src/main/kotlin/frc/robot/commandGroups/CommandGroupsConstants.kt

```diff
@@ -0,0 +1,24 @@
+package frc.robot.commandGroups
+
+import com.pathplanner.lib.util.GeometryUtil
+import edu.wpi.first.math.geometry.Translation2d
+import edu.wpi.first.units.Angle
+import edu.wpi.first.units.Measure
+import edu.wpi.first.units.Units
+import edu.wpi.first.units.Velocity
+import frc.robot.Constants
+
+object CommandGroupsConstants {
```

Comment:
This class holds only 🦸‍♂️💩 related constants. Why not just put all the relevant code in the same file? It's not very good to put all the command groups in one file anyway...

Reply:
c2fba2bb46f1fa4a0226f6ac6b20f1e04888d386